omnigibson/examples/object_states/attachment_demo.py

```diff
@@ -1,335 +1,112 @@
-    env, obj1, obj2 = setup_scene_for_abilities(
-        abilities1={"attachable": {"attachment_type": AttachmentType.SYMMETRIC, "attachment_category": "magnet"}},
-        abilities2={"attachable": {"attachment_type": AttachmentType.SYMMETRIC, "attachment_category": "magnet"}})
-    assert Attached in obj1.states
-    assert Attached in obj2.states
-
-    # Obj1 moves towards obj2 and they are attached together.
-    obj1.set_linear_velocity(velocity=np.array([3.0, 0, 3.0]))
-    for i in range(100):
-        env.step(np.array([]))  # empty action array since action space is 0 (no robots in the env)
-    assert obj1.states[Attached].get_value(obj2)
-    assert obj2.states[Attached].get_value(obj1)
-
-    # Apply a large force to obj1 but the two objects cannot move much because obj2 is heavy.
-    obj1.set_linear_velocity(velocity=np.array([10.0, 0, 50.0]))
-    for i in range(100):
-        env.step(np.array([]))  # empty action array since action space is 0 (no robots in the env)
-
-    # Unattach obj1 and obj2.
-    obj1.states[Attached].set_value(obj2, False)
-    assert not obj1.states[Attached].get_value(obj2)
-    assert not obj2.states[Attached].get_value(obj1)
-
-    # Obj1 moves away from obj2.
-    obj1.set_linear_velocity(velocity=np.array([-2.0, 0, 1.0]))
-    for i in range(100):
-        env.step(np.array([]))  # empty action array since action space is 0 (no robots in the env)
-
-
-def demo_failed_symmetric_attachment_missing_symmetric():
-    ######################################################################################
-    # Symmetric attachment - FAIL because only 1 object has the symmetric attachment type
-    #   can attach if touching and both objects have the symmetric attachment type and the same attachment category
-    ######################################################################################
-    env, obj1, obj2 = setup_scene_for_abilities(
-        abilities1={"attachable": {"attachment_type": AttachmentType.SYMMETRIC, "attachment_category": "magnet"}},
-        abilities2={})
-    assert Attached in obj1.states
-    assert Attached not in obj2.states
-
-    # Obj1 moves towards obj2 but they are NOT attached together.
-    obj1.set_linear_velocity(velocity=np.array([3.0, 0, 3.0]))
-    for i in range(100):
-        env.step(np.array([]))  # empty action array since action space is 0 (no robots in the env)
-    assert not obj1.states[Attached].get_value(obj2)
-
-    # Obj1 moves away from obj2.
-    obj1.set_linear_velocity(velocity=np.array([-2.0, 0, 1.0]))
-    for i in range(100):
-        env.step(np.array([]))  # empty action array since action space is 0 (no robots in the env)
+    for _ in range(10):
+        env.step([])
 
+    shelf_baseboard = og.sim.scene.object_registry("name", "shelf_baseboard")
+    shelf_baseboard.set_position_orientation([0, -0.979, 0.26], [0, 0, 0, 1])
+    shelf_baseboard.keep_still()
+    shelf_baseboard.set_linear_velocity([-0.2, 0, 0])
```

Comment:
what's the goal of this demo? If it's for the user to interact there should probably be explicit instructions printed into the terminal, otherwise we should probably inform the user what is happening and then execute the attach / detach programmatically

Reply:
Added an explicit instruction!